lib/manager/pip_setup/extract.ts

```diff
@@ -51,7 +51,8 @@ export async function extractSetupFile(
 ): Promise<PythonSetup> {
   const cwd = config.localDir;
   let cmd: string;
-  const args = [`"${join(__dirname, 'extract.py')}"`, `"${packageFile}"`];
+  const file = await resolveFile('data/extract.py');
```

Comment:
Are we still sure about `data/` ?

Reply:
Are you suggesting an other folder? Currently I'm expecting files there

Reply:
Or you you like to have them in manager folders?

Reply:
Seemed strange to have just one file in `data/` until now. And now I'm not sure that `extract.py` is a good fit for "data". Other than that I don't have an opinion.

Reply:
We have two files there, the nodejs schedule json!

Reply:
ok, let's keep it as-is